app/common/metrics.py

```diff
@@ -17,6 +20,23 @@
     'serialized_build_time_seconds',
     'Total amount of time that would have been consumed by builds if all work was done serially')
 
+internal_errors = Counter(
+    'internal_errors',
+    'Total number of internal errors',
+    ['type'])
+
+
+class ErrorType(str, Enum):
+    AtomizerFailure = 'AtomizerFailure'
+    NetworkRequestFailure = 'NetworkRequestFailure'
+    PostBuildFailure = 'PostBuildFailure'
+    SetupBuildFailure = 'SetupBuildFailure'
+    SubjobWriteFailure = 'SubjobWriteFailure'
+    ZipFileCreationFailure = 'ZipFileCreationFailure'
+
+    def __str__(self):
+        return self.value
```

Comment:
It's so that the /metrics endpoint has metrics that look like
`internal_errors{type="PostBuildFailure"} 1.0`
instead of 
`internal_errors{type="ErrorType.PostBuildFailure"} 1.0`

I don't think I need to inherit from `str`, i'm just doing it because all the other enums in cluster runner do it.

I'll add a comment.